quantum/keymap_common.c

```diff
@@ -74,7 +74,21 @@ action_t action_for_key(uint8_t layer, keypos_t key)
     } else if (keycode == RESET) { // RESET is 0x5000, which is why this is here
         clear_keyboard();
         _delay_ms(250);
+// bootloader_jump doesn't work for the Teensy
+#ifdef TEENSY
```

Comment:
I found this code to be a straight copy of the ´#ifdef __AVR_ATmega32U4__´ in ´bootloader_jump()´ in [tmk_core/common/avr/bootloader.c](https://github.com/jackhumbert/qmk_firmware/blob/master/tmk_core/common/avr/bootloader.c). As of such these changes can be reduced to what's defined in PR #162.